packages/kolibri-logging/src/index.js

```diff
@@ -23,10 +33,12 @@ class Logger {
 
   setMessagePrefix() {
     var originalFactory = this.logger.methodFactory;
+    var self = this;
```

Comment:
Could you pass an arrow-function on 37 instead of a `function()` and have the `this` you want on 39 in that scope?

Reply:
Or could you do something like `this.logger.methodFactory = function() {}.bind(this)` and reference `this.logColors` inside the function?

This thread is very much non-blocking just got me thinking

Reply:
Probably - I just kept this all very ES5.